### DISCUSSION TITLE
Takeoff c172x

### DISCUSSION THREAD

**Discussion by mosheFuks** [JSBSim-Team/jsbsim#789] [Q&A]
Hello, Im trying to do a takeoff simulation using the c172x model in Python, but im facing an issue. When I try to make the model takeoff the velocity gets lower and it never takesoff. For reference im using the script c172_elevation_test.
[here](https://gist.github.com/mosheFuks/ae6f3c46a13e9741cb1f05ebce417884) is my script, what im doing wrong?

**Comment by seanmcleod** [comment]
I don't think the engine is actually running. You could double-check by printing out the engine thrust to double-check that there is any thrust from the engine.

So to ensure that the engine is started and running I added the following to the initialisation file, the `-1` basically means all engines, useful for multi-engine aircraft.

```xml
<running> -1 </running>
```

Note that your loop with `dt = 0.008` means it only executes for 8s, so there is no way even with a running engine that it will take off in that time.

```python
for i in range(0,1000,1):
    simulation.fdm.run()
```

With the change to the initialisation file you can see that the velocity is now increasing over the 8s.

```sh
10.527164155744295
altura: 30.32660260390546,-20.018076438654216,1.3173593177855016
1.0
Freno 0.0
10.53828105887381
altura: 30.365716761386416,-20.092709594476872,1.3173677202522756
1.0
Freno 0.0
10.54939488525705
altura: 30.404725456627677,-20.16749823400843,1.3173761352092028
1.0
Freno 0.0
10.560505635662402
altura: 30.443628199307994,-20.242442216977253,1.3173845683336258
1.0
Freno 0.0
10.571613311488916
altura: 30.482424496983455,-20.317541405572634,1.3173930150836708
1.0
Freno 0.0
```
